src/BDoc.cpp

```diff
@@ -426,6 +433,15 @@ void BDoc::parseManifestAndLoadFiles(const ZipSerialize &z, const vector<string>
                 THROW("Found multiple references of file '%s' in zip container.", iter->full_path().c_str());
 
             manifestFiles.insert(iter->full_path());
+            if(mediaType() == BDoc::ADOC_MIMETYPE &&
```

Comment:
By definition ADOC is not BDOC. Maybe such constants could be in some more generic namespace ?